### DISCUSSION TITLE
'entity-per-chunk-save-limit' option and the new boat entity ID variant in 1.21.2+

### DISCUSSION THREAD

**Discussion by Huge-mistake** [PaperMC/Paper#11684] [Ideas]
### Is your feature request related to a problem?

Starting from version 1.21.2, Mojang has created unique entity IDs for each type of boat material. If I want to limit a chunk to save a maximum of 128 boats, it is currently impossible to do so because each type of boat has its own unique entity ID. It is clearly impractical to allocate only a dozen or so quotas for the entity IDs of each type of boat.

### Describe the solution you'd like.

Support using "AbstractBoat" or "boat and chest_boat" to cover all types of boats.

  entity-per-chunk-save-limit:
    abstractboat: 128
 #or
    boat: 128
    chest_boat: 128

### Describe alternatives you've considered.

Combine multiple entity IDs to share a limit on the number of entities？

  entity-per-chunk-save-limit:
    'oak_boat, spruce_boat, birch_boat, ..., bamboo_chest_raft': 128

### Other

https://docs.papermc.io/paper/reference/world-configuration#chunks_entity_per_chunk_save_limit

**Comment by masmc05** [comment]
Maybe supporting tags there could be useful, so you could just use `#boat` to add all boats

**Reply by Huge-mistake** [reply]
> Maybe supporting tags there could be useful, so you could just use `#boat` to add all boats

yeah, but I'm not sure if there's a correlation between the entity ID and the item tags.
